packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -697,6 +697,9 @@ abstract class WKWebViewHostApi {
 
   @ObjCSelector('setInspectableForWebViewWithIdentifier:inspectable:')
   void setInspectable(int identifier, bool inspectable);
+
+  @ObjCSelector('userAgentForWebViewWithIdentifier:')
```

Comment:
This should be `customUserAgentForWebViewWithIdentifier`, since it doesn't give you the actual user agent.